### PR TITLE
Implemented DataFrame.pad & Series.pad

### PR DESCRIPTION
**Pull request by itholic**
This PR proposes `DataFrame.pad` and `Series.pad`

This is basically synonym for `DataFrame.fillna` and `Series.fillna` with `method='ffill'` or `ffill` function

```python
>>> df = ks.DataFrame({
...     'A': [None, 3, None, None],
...     'B': [2, 4, None, 3],
...     'C': [None, None, None, 1],
...     'D': [0, 1, 5, 4]
...     },
...     columns=['A', 'B', 'C', 'D'])
>>> df
     A    B    C  D
0  NaN  2.0  NaN  0
1  3.0  4.0  NaN  1
2  NaN  NaN  NaN  5
3  NaN  3.0  1.0  4

>>> df.pad()
     A    B    C  D
0  NaN  2.0  NaN  0
1  3.0  4.0  NaN  1
2  3.0  4.0  NaN  5
3  3.0  3.0  1.0  4

>>> s = ks.Series([np.nan, 2, 3, 4, np.nan, 6], name='x')
>>> s
0    NaN
1    2.0
2    3.0
3    4.0
4    NaN
5    6.0
Name: x, dtype: float64

>>> s.pad()
0    NaN
1    2.0
2    3.0
3    4.0
4    4.0
5    6.0
Name: x, dtype: float64
```

### CODE REVIEW

File: databricks/koalas/generic.py [hunk 1]
```diff
@@ -2465,6 +2465,61 @@ def ffill(self, axis=None, inplace=False, limit=None):
...
+        >>> s.pad()  # doctest: +SKIP
+        0    NaN
+        1    2.0
+        2    3.0
+        3    4.0
+        4    4.0
+        5    6.0
+        Name: x, dtype: float64
+        """
+        return self.fillna(inplace=inplace, method="ffill")
```
Comment (HyukjinKwon) [inline databricks/koalas/generic.py:2521]: Can you add another alias `ffill`?

**Comment by HyukjinKwon** [comment]
Looks good otherwise.

**Comment by itholic** [comment]
Thanks, @HyukjinKwon !

**Comment by itholic** [comment]
I am pretty sure about this is okay to go since it's just synonym.

Please feel free to any comments !! :)

**Pull request merged**